draft-ietf-lamps-kyber-certificates.md

```diff
@@ -279,7 +279,9 @@ algorithm itself.
 </aside>
 
 When used in a OneAsymmetricKey type, the privateKey OCTET STRING contains
-the raw octet string encoding of the 64-octet seed.
+the raw octet string encoding of the 64-octet seed. The publicKey field
+SHOULD be omitted because the public key can be compute as noted earlier
```

Comment:
Typo: "computed". I'd phrase it like "because the public key is computed as part of using the private key".